scss/grid.scss

```diff
@@ -0,0 +1,89 @@
+.flex-body { 
+  grid-template-areas: 
+    "header"
+    "main"
+    "footer";
+
+  .train-header {
+    align-items: center;
+    display: grid;
+    grid-area: header;
+    grid-template-areas: 
+      "logo . dev";
+    top: 0;    
+
+    .logo {
+      grid-area: logo;
+    }
+
+    .dev {
+      grid-area: dev;
+      justify-self: right;
+    }
+  }
+
+  .train-main {
+    grid-area: main;
+    
+    .section .techs {
+      display: grid;
+      gap: 0;
+      grid-auto-flow: row;
+      grid-template-columns: repeat(3, 1fr);
+      grid-template-rows: auto;
+      place-items: center;
+      position: relative;
+
+      .tech {
+        background-color: transparent;
+
+        &::before {
+          border: none;
+        }
+
+        &:first-child {
+          grid-row: 1 / 3;
+          position: static;
+
+          &::after {
+            background-color: $training-green;
+            content: '';
+            grid-column: 1 / 2;
+            grid-row: 1 / 3;
+            height: 100%;
+            position: absolute;
+            right: 0;
+            top: 0;
+            width: 2px;
+          }
+        }
+      }      
+    }
+  }
+
+  .train-footer {
+    align-items: center;
+    display: grid;
+    grid-area: footer;
+    grid-template-areas: 
+      ". txt ."
```

Comment:
I dont like using this dots. Use align-items in parent and align-self in copyright